webpack.config.js

```diff
@@ -12,10 +13,21 @@ module.exports = {
     },
     module: {
         loaders: [
-            { test: /\.css$/,  loader: 'style!css' },
+            {
+                test:   /\.css$/,
+                loader: "style!css!postcss-loader"
+            },
             { test: /\.html$/, loader: 'html' },
         ]
     },
+    postcss: function () {
+        return [
+          prefix({
+            prefix: '.SGMAPembauche ', // <--- notice the trailing space!
+            exclude: ['.SGMAPembauche'] // in order to style the prefix itself in  the .css
```

Comment:
Add inner spaces for inline arrays and objects to make the data structure stand out more.